docs/src/stories/components/Dialog/Dialog.stories.jsx

```diff
@@ -0,0 +1,388 @@
+        category: 'Demo'
+      }
+    },
+    showCloseButton: {
+      control: {type: 'boolean'},
+      description: 'show/hide close button',
+      defaultValue: true,
+      table: {
+        category: 'Demo'
+      }
+    },
+    showFooterButton: {
+      control: {type: 'boolean'},
+      description: 'show/hide footer button',
+      defaultValue: false,
+      table: {
+        category: 'Demo'
+      }
+    },
+    width: {
+      options: [0, 1, 2, 3, 4, 5], // iterator
+      mapping: [
+        'Overlay--width-auto',
+        'Overlay--width-small',
+        'Overlay--width-medium',
+        'Overlay--width-large',
+        'Overlay--width-xlarge',
+        'Overlay--width-xxlarge'
+      ], // values
+      control: {
+        type: 'inline-radio',
+        labels: ['auto', 'xsmall', 'small', 'medium', 'large', 'xlarge', 'xxlarge']
+      },
+      description: 'Width options: small: 256px, medium: 320px, large: 480px, xlarge: 640px, xxlarge: 960px',
+      table: {
+        category: 'CSS'
+      }
+    },
+    height: {
+      options: [0, 1, 2, 3, 4, 5], // iterator
+      mapping: [
+        'Overlay--height-auto',
+        'Overlay--height-xsmall',
+        'Overlay--height-small',
+        'Overlay--height-medium',
+        'Overlay--height-large',
+        'Overlay--height-xlarge'
+      ], // values
+      control: {
+        type: 'inline-radio',
+        labels: ['auto', 'xsmall', 'small', 'medium', 'large', 'xlarge', 'xxlarge']
+      },
+      description:
+        'Height options: auto: adjusts to content, xsmall: 192px, small: 256px, medium: 320px, large: 432px, xlarge: 600px',
+      table: {
+        category: 'CSS'
+      }
+    },
+    headerVariant: {
+      options: [0, 1], // iterator
+      mapping: ['', 'Overlay-header--large'], // values
+      control: {
+        type: 'inline-radio',
+        labels: ['medium (default)', 'large']
+      },
+      description: 'medium (default), large header/description font-size + spacing',
+      table: {
+        category: 'CSS'
+      }
+    },
+    bodyPaddingVariant: {
+      options: [0, 1, 2], // iterator
+      mapping: ['', 'Overlay-body--paddingCondensed', 'Overlay-body--paddingNone'], // values
+      control: {
+        type: 'inline-radio',
+        labels: ['normal (default)', 'condensed', 'none']
+      },
+      description: 'body spacing',
+      table: {
+        category: 'CSS'
+      }
+    },
+    variantNarrow: {
+      options: [0, 1, 2, 3], // iterator
+      mapping: [
+        'Overlay-backdrop--center-whenNarrow',
+        'Overlay-backdrop--anchor-whenNarrow',
+        'Overlay-backdrop--side-whenNarrow',
+        'Overlay-backdrop--full-whenNarrow'
+      ], // values
+      control: {
+        type: 'inline-radio',
+        labels: ['center', 'anchored', 'side', 'full']
+      },
+      description: '',
+      table: {
+        category: 'Variant'
+      }
+    },
+    variantRegular: {
+      options: [0, 1, 2, 3], // iterator
```

Comment:
Pro-tip™: you can pass API-friendly values directly to the `options` object.

[Example](https://github.com/primer/css/pull/1980/files#diff-5e3ad3b2f2fad136b5d249ee4d9b74131f9394fc282b71fe34dbec8d72ed067bR64):

```
options: ['title-large', 'title-medium', 'subtitle'],
```